.github/workflows/ci.yml

```diff
@@ -1,434 +1,453 @@
-      - name: Run tests
-        run: ./ci/model_management_integration_tests.sh
-      - name: Upload test coverage to Codecov
-        uses: codecov/codecov-action@v1.0.12
-
-  yatai_server_integration_tests:
-    runs-on: ubuntu-latest
-    services:
-      # Label used to access the service container
-      postgres:
-        # Docker Hub image
-        image: postgres
-        # Provide the password for postgres
-        env:
-          POSTGRES_PASSWORD: postgres
-        # Set health checks to wait until postgres has started
-        options: >-
-          --health-cmd pg_isready
-          --health-interval 10s
-          --health-timeout 5s
-          --health-retries 5
-        ports:
-          # Maps tcp port 5432 on service container to the host
-          - 5432:5432
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0  # fetch all tags and branches
-      - name: Setup python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.7
-      - name: Install test dependencies
-        run: ./ci/install_test_deps.sh
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-      - name: Run tests
-        run: ./ci/yatai_server_integration_tests.sh
-      - name: Upload test coverage to Codecov
-        uses: codecov/codecov-action@v1.0.12
-
-  s3_bucket_integration_test:
-    runs-on: ubuntu-latest
-    services:
-      minio:
-        image: yubozhao/minio-github-action-service
-        env:
-          MINIO_ACCESS_KEY: bentoml
-          MINIO_SECRET_KEY: bentoml
-        ports:
-          - 9000:9000
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0  # fetch all tags and branches
-      - name: Setup python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.7
-      - name: Install test dependencies
-        run: ./ci/install_test_deps.sh
-      - name: Run s3 test
-        run: ./ci/s3_bucket_integration_tests.sh
-        env:
-          AWS_ACCESS_KEY: bentoml
-          AWS_SECRET_ACCESS_KEY: bentoml
-      - name: Upload test coverage to Codecov
-        uses: codecov/codecov-action@v1.0.12
-
-  pytorch_lightning_integration_tests:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-        with:
-          fetch-depth: 0  # fetch all tags and branches
-      - name: Setup python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.7
-      - name: Install test dependencies
-        run: ./ci/install_test_deps.sh
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
-      - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-      - name: Run tests
-        run: ./ci/pytorch_lightning_integration_tests.sh
-      - name: Upload test coverage to Codecov
-        uses: codecov/codecov-action@v1.0.12
+name: BentoML-CI

```

Comment:
These lines are unnecessarily marked as modified. I guess this file was saved with the Windows=-style newline `\r\n`, right?

Reply:
That might be the case, I'll fix this